socorro/processor/rules/memory_report_extraction.py

```diff
@@ -82,15 +86,21 @@ def _get_memory_measures(self, memory_report, pid):
         # These ones are derived from the memory report.
         metrics_derived = {
             'explicit': 0,
-            'heap-overhead': 0,
-            'heap-unclassified': 0,
+            'heap_overhead': 0,
+            'heap_unclassified': 0,
             'images': 0,
-            'js-main-runtime': 0,
-            'top-non-detached': 0,
+            'js_main_runtime': 0,
+            'top_non_detached': 0,
         }
 
         all_metrics = {}
-        all_metrics.update(metrics_measured)
+        # Replace the dashes in keys with underscores for consistency.
```

Comment:
Just do this step at the end, and then all the other changes to this script won't be necessary. That way the script can uses dashes all throughout, which fits with how memory reporting works, and then we switch to underscores in a single place.